test/compilable/imports/cstuff1.c

```diff
@@ -440,4 +440,6 @@ int main()
 #line 1000
 %:line 1010
 
+# 1020 "cstuff1.c" 1 2 3 4
+# 1030
```

Comment:
Adding a test with a line marker in the middle of a struct-declaration-list would be good, to prove that the referenced bug is indeed fixed.

Reply:
??

Reply:
Adapted from the bug report 
```
struct S21944
{
    int var;
#1040 "cstuff1.c" 3 4
}; 
```
This is what triggered the error.

Reply:
ok